### PR TITLE
Add `max_frames` + `max_output_size` arguments for Aria data processing

### PR DESCRIPTION
**Pull request by brentyi**
@AntonioMacaronio this has been helpful for some of datasets recently. Does it make sense to you? ( I know there's also the other open Aria + data processing PRs )

### CODE REVIEW

**Comment by AntonioMacaronio** [comment]
These changes make sense to me, but I am wondering about the fisheye_crop_radius

If a user limits the resolution of the image to be smaller than 1404, the fisheye_crop_radius would have to decrease proportionally to their new resolution right?

**Comment by brentyi** [comment]
> If a user limits the resolution of the image to be smaller than 1404, the fisheye_crop_radius would have to decrease proportionally to their new resolution right?

Yeah, the code is a little bit hard to follow but since I'm updating the camera calibration based on the new image resolution this does end up getting handled correctly!
